### PR TITLE
Added Andrew to authorized users

### PR DESCRIPTION
**Pull request by AndrewKahr**
#### Changes

- ...

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [x] Readme (updated or not needed)
- [x] Tests (added, updated or not needed)

### CODE REVIEW

**Review by GabLeRoux — APPROVED** [review]

**Comment by GabLeRoux** [comment]
small linter issue, feel free to merge once fixed 👍

**Comment by AndrewKahr** [comment]
@GabLeRoux How do I run format in this repo? I don't see yarn set up nor do I see a format call via npm

**Comment by AndrewKahr** [comment]
Nvm I've added an additional config to run format with --write

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "lint": "prettier --check \"functions/src/**/*.{js,jsx,ts,tsx}\"",
+    "format": "prettier --write \"functions/src/**/*.{js,jsx,ts,tsx}\"",
```
Comment (webbertakken) [inline package.json:8]: This should generally be integrated in your editor. While I have nothing against adding this, you shouldn't need to run this manually.

**Pull request merged**